e2e-tests/native/test/pageobjects/page.ts

```diff
@@ -0,0 +1,13 @@
+
+export class Page {
+    readonly pageId
+
+    constructor(pageId: string) {
+        this.pageId = pageId
+    }
+
+    public async exists(): Promise<boolean | void> {
```

Comment:
Better return type would be just Promise<boolean>. Can we map void to boolean here?